modules/core/src/main/java/org/apache/ignite/configuration/DataStorageConfiguration.java

```diff
@@ -252,6 +253,14 @@ public class DataStorageConfiguration implements Serializable {
     @IgniteExperimental
     private long cdcWalDirMaxSize = DFLT_CDC_WAL_DIRECTORY_MAX_SIZE;
 
+    /** Maximum size of CDC buffer. */
+    @IgniteExperimental
+    private long maxCdcBufSize = (long)DFLT_WAL_SEGMENTS * DFLT_WAL_SEGMENT_SIZE;
+
+    /** CDC buffer consumer. */
+    @IgniteExperimental
+    private transient CdcBufferConsumer cdcConsumer;
```

Comment:
We shouldn't add internal interface in public API methods.

Reply:
Right now, CDC is enabled only for some data regions. 
It will be more convinient to have `CdcBufferConsumer` near the `cdcEnabled` flag.

Right now, it's not clear what is enabled if `DataStorageConfiguration#cdcConsumer` set.